packages/nimble-components/src/rich-text-editor/tests/rich-text-editor.stories.ts

```diff
@@ -103,11 +127,46 @@ const metadata: Meta<RichTextEditorArgs> = {
         },
         setMarkdownData: {
             table: { disable: true }
+        },
+        errorVisible: {
+            description:
+                'Whether the editor should be styled to indicate that it is in an invalid state.'
+        },
+        errorText: {
+            description:
+                'A message to be displayed when the editor is in the invalid state explaining why the value is invalid.'
+        },
+        placeholder: {
+            description: 'Placeholder text to show when editor is empty.'
+        },
+        footerHidden: {
+            description:
+                'Setting `footer-hidden` hides the footer section which consists of all formatting option buttons and the `footer-actions` slot content.'
+        },
+        fitToContent: {
+            description: fitToContentDescription
+        },
+        empty: {
+            name: 'empty',
+            description:
+                'Read-only boolean value. Returns true if editor is empty.',
```

Comment:
We should explicitly say this includes when the editor has only whitespace (assuming that's the behavior we want).

Reply:
Done